tools/gen-cmodules.py

```diff
@@ -6,22 +6,29 @@
 
```

Comment:
I'm not sure commit message here explains purpose, format, usage of cmodule.json well enough.